### PR TITLE
completed unique-url and success page

### PR DESCRIPTION
**Pull request by tjw0612**
## Why

This pull request is needed because:

New gem is added
Installation required
Migration required

There is now unique link 
Able to create new Chomp Session from start to finish
Able to join existing Chomp Session

## What

This pull request introduces the following:

 * `Changes introduced`

`Add Implementation details if needed`

### Screenshot

`Attach any screenshots here. Remove when unneeded`

## How to Test

`Give the reviewer(s) some clues on how to test your PR`

## Other

`This section is for things that can't be put into the previous sections. Remove when unneeded`

### CODE REVIEW

File: app/views/chomp_sessions/success.html.erb [hunk 1]
```diff
@@ -1 +1,19 @@
-<h1 class="test-font">Success!</h1>
\ No newline at end of file
+<div class="container d-flex flex-column justify-content-center my-5 w-50 p-5" style="background-color:white">
+
+<h1 class="success-header">Group created</h1>
+<p class="success-body">Congratulations! Your group has been created, share this link with you friends and get started on planning your next trip!</p>
```
Comment (mxng) [inline app/views/chomp_sessions/success.html.erb]: can have some indentation/formatting? 👀

**Review by mxng — CHANGES_REQUESTED** [review]
Looks ok! Except one file seems to have no indentation?

File: app/views/chomp_sessions/success.html.erb (hunk 1 above)
Comment (tjw0612) [reply app/views/chomp_sessions/success.html.erb]: indentation added in new commit

**Review by mxng — APPROVED** [review]
👍

**Pull request merged**